api/applications/serializers.py

```diff
@@ -37,12 +39,13 @@ def get_absolute_url(self, obj):
         return obj.get_absolute_url()
 
     def reset_url(self, obj):
-        return absolute_reverse(
-            'applications:application-reset', kwargs={
-                'client_id': obj.client_id,
-                'version': self.context['request'].parser_context['kwargs']['version'],
-            },
-        )
+        if StrictVersion(self.context['request'].version) < StrictVersion('2.13'):
+            return absolute_reverse(
+                'applications:application-reset', kwargs={
+                    'client_id': obj.client_id,
+                    'version': self.context['request'].parser_context['kwargs']['version'],
+                },
+            )
```

Comment:
👍 reset_url only returned in early versions (here post-2.13, but will need to be updated w/ the latest version).  Later versions, reset_url will not be serialized with the application at all.